src/drivers/distance_sensor/vl53l1x/vl53l1x.cpp

```diff
@@ -222,35 +232,55 @@ void VL53L1X::RunImpl()
 {
 	uint8_t dataReady = 0;
 
+	uint8_t roiCenter[] = {247, 215, 183, 167, 151}; 
+        static uint8_t zone = 0;
+
 	VL53L1X_CheckForDataReady(&dataReady);
 
 	if (dataReady == 1) {
-		collect();
+                collect();
```

Comment:
Whitespace / indentation change that should be removed (wrong indentation and it adds noise to the diff)